platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/version/ActivateWorkflowVersionAction.java

```diff
@@ -43,17 +46,23 @@ public IDisplayString getTitle() {
 	}
 
 	@Override
-	public void handleClick(AGWorkflowVersion tableRow) {
+	public void handleClick(AGWorkflowVersion version) {
 
 		try (DbTransaction transaction = new DbTransaction()) {
-			tableRow//
-				.setDraft(false)
-				.saveIfNecessary();
-			tableRow//
-				.getWorkflow()
-				.setCurrentVersion(tableRow)
-				.save();
-			transaction.commit();
+			version.reloadForUpdate();
```

Comment:
Is this merely done to acquire an x-lock? If so, we should probably do that explicitly. If not, we don't we need to reload all records that belong to that version? (nodes, etc)